sandbox/handlers/utils.js

```diff
@@ -37,7 +37,24 @@ function sendError(res, code, message) {
   });
 }
 
+function hasValidGlobalTemplatePersonalisation(personalisation) {
+    if (!personalisation) {
+      return false;
+    }
+  
+    const personalisationFields = Object.keys(personalisation);
+    if (personalisationFields.length != 1) {
+      return false;
+    }
+  
+    if (personalisationFields[0] !== "body") {
```

Comment:
Should we not just be checking if personalisationFields length is greater than 0 and includes `"body"`?

Reply:
I'm also on the fence of it strictly checking for "body" when this might change between clients

Reply:
Oh, I see this is for the sandbox environment so presumably this is for a dummy request... nevermind

Reply:
This validation ensures that the personalisation is exactly `{ body: string }`, with `body` being mandatory and no extra redundant fields.  This aims to help sandbox integrators to correctly use the global routing config.